api/entries.js

```diff
@@ -4,6 +4,7 @@ var Q = require('q');
 
 // save entry helper
```

Comment:
Needs better function description. Ex: Creates promise used for tracking multi-step .... etc